src/phpDocumentor/ContainerDefinitions.php

```diff
@@ -160,6 +171,27 @@ class_alias(Extension::class, 'phpDocumentor\\Plugin\\Core\\Xslt\\Extension');
     DocumentationFactory::class => \DI\object()
         ->method('addDocumentGroupFactory', \DI\get(phpDocumentor\ApiReference\Factory::class)),
 
+    //ApiReference
+    phpDocumentor\ApiReference\Factory::class => function (ContainerInterface $c) {
+        $strategies = [
+            new Argument(new PrettyPrinter()),
```

Comment:
An improvement here could be to retrieve the service from the DIC using `\DI\get(Argument::class)` so that it will autoresolve the dependency and ensure that it can be defined in the DIC